presto-orc/src/test/java/com/facebook/presto/orc/TestingOrcPredicate.java

```diff
@@ -333,16 +338,24 @@ protected boolean chunkMatchesStats(List<String> chunk, ColumnStatistics columnS
 
             // statistics can be missing for any reason
             if (columnStatistics.getStringStatistics() != null) {
-                // verify min
                 Slice chunkMin = Ordering.natural().nullsLast().min(slices);
-                if (columnStatistics.getStringStatistics().getMin().compareTo(chunkMin) > 0) {
-                    return false;
-                }
-
-                // verify max
                 Slice chunkMax = Ordering.natural().nullsFirst().max(slices);
-                if (columnStatistics.getStringStatistics().getMax().compareTo(chunkMax) < 0) {
-                    return false;
+                if (format == DWRF) {
+                    // the old dwrf writer uses UTF-16be for string stats which are widened by the reader
```

Comment:
Add comment that says `format` is not the important distinction here. The distinction is version. And we are using old DWRF version and new ORC version.